go.mod

```diff
@@ -6,5 +6,5 @@ require (
 	github.com/beyondstorage/go-service-cos/v2 v2.0.0
 	github.com/beyondstorage/go-service-fs/v3 v3.0.0
 	github.com/beyondstorage/go-service-s3/v2 v2.0.0
```

Comment:
Services have implemented [GSP-87](https://github.com/beyondstorage/specs/blob/master/rfcs/87-feature-gates.md) but not bump to new version. Only update `go-storage` will causes compilation failure.

Reply:
Got it. Let's bump new version after create dir implemented.

Reply:
Does `go-integration-test` need to bump new version before services?